.gitignore

```diff
@@ -5,4 +5,6 @@ ibctest.test
 /bin
 
 #GoLand project directory
-.idea/
\ No newline at end of file
+.idea/
+
+.vscode/
```

Comment:
`.idea/` was likely my fault. 

These types of files are ideally in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c), so they aren't committed to the repo. So, best practices is delete `.idea` and `.vscode` here and instead add them to your global gitignore.

Reply:
Oh Nice!